docs/tutorials/data_manipulation/dataset_splitting_example.md.template

```diff
@@ -0,0 +1,118 @@
+# Splitting datasets into training and test
+
+In this notebook, we are going to show that given a dataset how can we split it when necessary. Specifically, we'll use the `split` function that already existed in our GluonTS project.
+
+In the `split` function:
+- you need to provide the `dataset` for the method to split.
+- you also need to provide `offset` or `date`, but not both simultaneously. These two arguments are provided for the method to know how to slices training and test data based on a fixed integer offset or a ``pandas.Period``.
+As a result, the `split` method returns the splited dataset, consisting of the training data `training_dataset` and the TestTemplate objectives `test_template` which knows how to generate test data `test_pairs` using the memeber function `generate_instances`.
+
+
+## Data loading and processing
+
+
+```python
+from gluonts.dataset.field_names import FieldName
+from gluonts.dataset.pandas import PandasDataset
+from gluonts.dataset.split.splitter import split
+from gluonts.dataset.util import to_pandas
+```
+
+
+```python
+%matplotlib inline
+import pandas as pd
+import matplotlib.pyplot as plt
+```
+
+Config plotting attributes.
+
+
+```python
+plt.rcParams["axes.grid"] = True
+plt.rcParams["figure.figsize"] = (20,3)
+```
+
+### Get some datasets
+
+Load data from an online csv into `pandas.DataFrame`, which is a long time series, with the frequency being 5 minutes and the length being 15831, so we resample the time series into an hour bins and sum the values of the timestamps falling into a bin, making the time series look nicer and more organized.
+
+
+```python
+url = "https://raw.githubusercontent.com/numenta/NAB/master/data/realTweets/Twitter_volume_AMZN.csv"
+df = pd.read_csv(url, header=0)
+df["timestamp"] = pd.to_datetime(df["timestamp"])
+df = df.set_index("timestamp")
+df_resample = df.resample("1H").sum()
+dataset = PandasDataset(df_resample.reset_index(), timestamp="timestamp", target="value")
```

Comment:
There's a log going on here, let's simplify this a bit:
- in `read_csv` you can pass `index_col="timestamp"` and `parse_dates=True` to directly get a `DatetimeIndex`
- we can directly `.resample` and `.sum` in the same line as `.read_csv`
- we can move the creation of the `PandasDataset` to later on, after we visualized the data

As a result, this snippet should be just 2 lines of code